base/operators/openshift-special-resource-operator/02-node-feature-discovery.yaml

```diff
@@ -0,0 +1,127 @@
+---
+apiVersion: nfd.openshift.io/v1
+kind: NodeFeatureDiscovery
```

Comment:
- This has dependency on the Node Feature Discovery (NFD) Operator. Can we include a README.md with that pointer?

Reply:
As of this writing, NFD is installed by SRO.  Let's add a separate PR to add NFD if someone wants to add it stand-alone.

Reply:
Okay. Thanks for the clarification here. Wondering how this is handled if NFD is already installed in the cluster. I'll put this one as an issue for testing